shared/versions/stack/master.asciidoc

```diff
@@ -13,7 +13,7 @@ bare_version never includes -alpha or -beta
 :prev-major-version:     7.x
 :prev-major-last:        7.17
 :major-version-only:     8
-:ecs_version:            master
+:ecs_version:            8.9
```

Comment:
Can someone from docs verify this is ok? 

We noticed that as a part of the [8.8 FF PR](https://github.com/elastic/docs/pull/2672) this was changed to master from the specific version. Previously, our team (ECS) has always set this ([8.7 FF PR](https://github.com/elastic/docs/pull/2669/files))

Reply:
Yeah, that looks like it indeed should be `8.9`. Nice catch!

Reply:
Awesome, thanks @kilfoyle !

Reply:
Let me check on this...

Reply:
Checked again. Yes indeed, it should be `8.9`. :-)